IFC4x3/PropertyEnumerations/t/PEnum_TurnoutHeaterUsed.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Documentation?

Reply:
erased. Redundancy due to IfcDoc. Commit : 426664c10f4b7a9f9b1eff4dddaa4ee62a65d0e3